make command fails on make swagger

I'm unable to make snap from surces:
make swagger
make[1]: Entering directory '/home/pmatyjas/go/src/github.com/intelsdi-x/snap'
bash -c "./scripts/swagger.sh"
./scripts/swagger.sh: line 22: swagger: command not found
./scripts/swagger.sh: line 23: swagger: command not found
Makefile:64: recipe for target 'swagger' failed
make[1]: *** [swagger] Error 127
make[1]: Leaving directory '/home/pmatyjas/go/src/github.com/intelsdi-x/snap'
Makefile:22: recipe for target 'default' failed
make: *** [default] Error 2

There is lack of documentation about swagger installation.
Snap revision:
git describe
1.2.0-83-g22cd662

--- a/.travis.yml
+++ b/.travis.yml
@@ -5,8 +5,8 @@ go:
 before_install:
 - bash scripts/gitcookie.sh
 - go get github.com/smartystreets/goconvey/convey
-- echo "deb https://dl.bintray.com/go-swagger/goswagger-debian ubuntu main" | sudo tee -a /etc/apt/sources.list
-- sudo apt-get update && sudo apt-get install -y swagger --force-yes
+- curl -LO https://github.com/go-swagger/go-swagger/releases/download/0.10.0/swagger_linux_amd64
+- chmod +x swagger_linux_amd64 && sudo mv swagger_linux_amd64 /usr/bin/swagger
 - if [ ! -d $SNAP_SOURCE ]; then mkdir -p $HOME/gopath/src/github.com/intelsdi-x; ln -s $TRAVIS_BUILD_DIR $SNAP_SOURCE; fi # CI for forks not from intelsdi-x
 env:
   global:

--- a/README.md
+++ b/README.md
@@ -84,6 +84,22 @@ The key features of Snap are:
 
 ### System Requirements
 
+Snap needs Swagger ([http://swagger.io](http://swagger.io)) installed to generate Swagger spec file during build process.
+
+To install Swagger:
+```sh
+echo "deb https://dl.bintray.com/go-swagger/goswagger-debian ubuntu main" | sudo tee -a /etc/apt/sources.list
+sudo apt-get update
+sudo apt-get install swagger
+```
+
+or from GitHub release:
+
+```sh
+curl -LO https://github.com/go-swagger/go-swagger/releases/download/0.10.0/swagger_linux_amd64
+chmod +x swagger_linux_amd64 && sudo mv swagger_linux_amd64 /usr/bin/swagger
+```
+
 Snap does not have external dependencies since it is compiled into a statically linked binary. At this time, we build Snap binaries for Linux and MacOS. We also provide Linux RPM/Deb packages and MacOS X .pkg installer.
 
 ### Installation

